### PR TITLE
Update README.md and remove deprecated actions

### PR DESCRIPTION
**Pull request by rberrelleza**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+# GitHub Actions for Okteto
 
-## Automate your development workflows using Github Actions and Okteto Cloud
-GitHub Actions gives you the flexibility to build an automated software development workflows. With GitHub Actions for Okteto Cloud you can create workflows to build, deploy and update your applications in [Okteto Cloud](https://cloud.okteto.com).
+## Automate your development workflows using GitHub Actions and Okteto
 
-Get started today with a [free Okteto Cloud account](https://cloud.okteto.com)!
+GitHub Actions gives you the flexibility to build automated software development workflows. With GitHub Actions for Okteto, you can create workflows to build, deploy and update your applications in [Okteto](https://okteto.com).
+
+Get started today [for a free Okteto account](https://okteto.com)!
```
Comment (omnomagonz) [inline README.md]:
```suggestion
Get started today [with a free Okteto account](https://okteto.com)!
```

Should we link to self-hosted here to be consistent with our current funnel/journey?

File: README.md [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
+
+Please try out the GitHub Actions for Okteto and share your feedback via Twitter on [@OktetoHQ](https://twitter.com/oktetohq). If you encounter a problem, please open an issue on the GitHub repository for the specific action.
```
Comment (omnomagonz) [inline README.md]:
```suggestion
Please try out the GitHub Actions for Okteto and share your feedback via Twitter with [@OktetoHQ](https://twitter.com/oktetohq). If you encounter a problem, please open an issue on the GitHub repository for the specific action.
```

File: README.md [hunk 3]
```diff
@@ -1,31 +1,32 @@
...
 
 # Available Actions
 
-- [Login](https://github.com/okteto/login)
+> All our actions work on [Okteto SaaS and Okteto Self-Hosted](https://www.okteto.com/pricing).
```
Comment (omnomagonz) [inline README.md]:
```suggestion
> All of our actions work on [Okteto SaaS and Okteto Self-Hosted](https://www.okteto.com/pricing).
```

**Review by omnomagonz — APPROVED** [review]
Non-blocking suggestions. Otherwise the PR looks good to me.

**Review by pchico83 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rberrelleza) [reply README.md]: Yes, good idea.

**Review by omnomagonz — APPROVED** [review]

**Pull request merged**